src/main/java/uk/nhs/tis/trainee/notifications/model/NotificationType.java

```diff
@@ -75,12 +77,34 @@ public enum NotificationType {
   @Getter
   private static final Set<NotificationType> programmeUpdateNotificationTypes = EnumSet.of(
       PROGRAMME_DAY_ONE,
+      PROGRAMME_UPDATED_WEEK_12,
       PROGRAMME_UPDATED_WEEK_8,
       PROGRAMME_UPDATED_WEEK_4,
+      PROGRAMME_UPDATED_WEEK_2,
       PROGRAMME_UPDATED_WEEK_1,
       PROGRAMME_UPDATED_WEEK_0,
       PROGRAMME_CREATED);
 
+  /**
+   * The set of currently active Programme Updated email notification types.
+   */
+  @Getter
+  private static final Set<NotificationType> activeProgrammeUpdateNotificationTypes = EnumSet.of(
```

Comment:
I don't think we can get rid of the others, since they might already be stored in some distant-future programme notification.